process/io/obsolete_vars.py

```diff
@@ -104,7 +104,10 @@
     "bcylth": None,
     "itfmod": None,
     "jcrit_model": None,
-    "jcritsc": None,
+    "jcritsc": "j_crit_sc",
+    "jcritstr": "j_crit_cable",
+    "jstrandoh_bop": "jcableoh_bop",
+    "jstrandoh_eof": "jcableoh_eof",
```

Comment:
These variables do not need to be added here because they are not input variables.

Reply:
Thanks, have removed.